app-admin/radmind/radmind-1.14.1-r1.ebuild

```diff
@@ -1,12 +1,12 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
```

Comment:
what happened?

Reply:
Huh.. haha.. long day i guess ;)
Thanks for pointing that out.. fixed

Reply:
just let repoman handle that for you, in fact